README.md

```diff
@@ -0,0 +1,58 @@
+The main reason why we use Linux is to have a similar environment to the Raspberry Pi.
```

Comment:
We should put a title for the readme overall

Reply:
Don't forget to turn this readme to an `.adoc` file

Reply:
Oh, I think we should also mention that the Raspberry Pi is used in the Gen 5 solar car. What do you think of this?

```
The main reason why the software team uses Linux is to have a similar environment to the Raspberry Pi, which Schulich Elysia's dashboard systems run on. 
As such, most projects made for Elysia will predominantly compile and work on Debian based linux systems.
```

Then drop the next line (2)